app/assets/stylesheets/building_map.scss

```diff
@@ -83,6 +83,26 @@
   content: '2';
 }
 
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 200px;
+  background-color: #ffffff;
+}
+
+li a {
```

Comment:
The same reasoning applies to styling all the `li`s, you can however reuse the fact that you styled the enclosing `ul` and only select `li`s inside it. One more thing: We have SCSS here, so you can use this convenient feature that let's you save on rewriting parent elements:

```scss
ul.my-class {
  // style for ul

  li a {
    // same as `ul.my-clas li a` outside of the block

    // style, same as your 2nd selector

    &:hover {
      // same as your 3rd selector
    }
  }
}
```